ZenPacks/zenoss/Import4/bin/imp4_check.sh

```diff
@@ -49,12 +49,7 @@ for i in \
         EVENTS_* \
         _zen* \
         componentList.txt \
-        zenbackup_*.tgz \
```

Comment:
zenbackup_<some tag> may be different in different export4 file. It may got left behind..